app/controllers/pulitzer/post_types_controller.rb

```diff
@@ -39,8 +41,19 @@ def post_type_params
     params[:post_type].permit!
   end
 
+  def post_params_name
+    params[:post_type][:name]
+  end
+
   def get_post_type
     @post_type = Pulitzer::PostType.find(params[:id])
   end
 
+  def create_singleton_post(post_type)
```

Comment:
I think this could be moved to a new interaction, what do you think @gomayonqui ?